axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceDiscountService.java

```diff
@@ -0,0 +1,17 @@
+package com.axelor.apps.account.service.invoice;
+
+import com.axelor.apps.account.db.Invoice;
+import com.axelor.apps.base.AxelorException;
+import java.math.BigDecimal;
+import java.util.Map;
+
+public interface InvoiceDiscountService {
```

Comment:
Please rename to `InvoiceGlobalDiscountService` to avoid confusion with other services that compute the discount on invoice lines.